.circleci/config.base.yml

```diff
@@ -1469,7 +1469,9 @@ commands:
                   source $BASH_ENV
             - run:
                 name: Confirm Amplify CLI is installed and available in PATH
-                command: amplify version
+                command: |
+                  which amplify
```

Comment:
Added this to confirm that `amplify` resolves right path. I find it useful for troubleshooting, hence leaving it.